### ISSUE TITLE
Nan Issue with STFTMagnitudeLoss

### ISSUE THREAD

**Issue by dlfqhsdugod1106** [csteinmetz1/auraloss#63]
Hello,
First of all, thanks for the very useful library.

I found that the additional epsilon term is missing in auraloss/freq.py line 47-48 (torch.log).
This part sometimes causes nan gradient.
I modified the code by my own in my environment, but it would be helpful if the official code is also fixed.

Thanks.
